modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/edit_kaleo_definition_version.jsp

```diff
@@ -332,109 +332,72 @@ String successMessageKey = KaleoDesignerPortletKeys.KALEO_DESIGNER + "requestPro
 								</div>
 
 								<aui:script>
-									Liferay.provide(
-										window,
-										'<portlet:namespace />afterTabViewChange',
-										function (event) {
-											var tabContentNode = event.newVal.get('boundingBox');
+									function <portlet:namespace />afterTabViewChange(event) {
```

Comment:
This feels risky, letting `window` be polluted because the script is not inside a closure. We've been assigning this directly to make sure intent is clear and won't break for example if we do `aui:script ="sandbox"` or anything else. As is, this is now, I don't think it's clear that this is expected to be a public API on `window`.

Reply:
> As is, this is now, I don't think it's clear that this is expected to be a public API on window.

You're right, it isn't clear at all. 
I actually didn't understand that the intent was to make this be a public API on window - but you've made it clear. 
I'll update that. Thanks.